p4sdk/images/entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# Copyright 2024 Intel Corp. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+export SDE_INSTALL=/opt/p4/p4sde
+export P4CP_INSTALL=/opt/p4/p4-cp-nws
+export DEPEND_INSTALL=$P4CP_INSTALL
+export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
```

Comment:
> Is this variable used?

This is needed when we use `$P4CP_INSTALL/bin/p4rt-ctl` otherwise I have seen python protobuf incompatibility errors popping up.

Reply:
Maybe it was it an older CI-build - could try to remove and unset this environment variable and paste a copy of the log errors?

Reply:
We are going to keep this for now. Can be removed in future when we know for sure this isn't required.